### PR TITLE
REF move a single benchmark to python

### LINKED ISSUE #602: setup example benchmark in python by converting one of them

### PR DESCRIPTION
**Pull request by beckermr**
This PR moves one benchmark to python. 

Closes #602

### CODE REVIEW

**Comment by beckermr** [comment]
@philbull care to approve this?

**Comment by damonge** [comment]
Did you also intend to remove the C version of this? Or are we gonna duplicate all tests?

**Comment by beckermr** [comment]
I don't plan to remove the C versions yet. We will have to start, but I wanted to get this working and done first and then clean out a bunch all at once. We can remove it now though if that is better.

**Comment by damonge** [comment]
No, that's fine, just wanted to understand the logic

File: benchmarks/data/model1_hmf.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#[0]log10[M/Msun] [1]sigma [2]log10[1/sigma] [3]log10[M^2/rho_m*dn/dM]
```
Comment (damonge) [inline benchmarks/data/model1_hmf.txt]: where did this file come from? Is it the same one being used in C?

Comment (beckermr) [reply benchmarks/data/model1_hmf.txt]: I copied the C file. I actually could not find the code that generated this benchmark.

Comment (damonge) [reply benchmarks/data/model1_hmf.txt]: No, that's fine. Sounds good.

Comment (damonge) [reply benchmarks/data/model1_hmf.txt]: I just didn't remember the original file having log(1/sigma)

**Pull request merged**